src/js/App/Sidenav/SideNav.js

```diff
@@ -8,21 +7,26 @@ import { appNavClick } from '../../redux/actions';
 import NavLoader from './Loader';
 import './SideNav.scss';
 import { isFilterEnabled } from '../../utils/isAppNavEnabled';
+import { globalNavComparator } from '../../utils/comparators';
 
 export const SideNav = () => {
   const dispatch = useDispatch();
-  const { activeTechnology, globalNav, appId } = useSelector(({ chrome }) => chrome);
-  const [isFirst, setIsFirst] = useState(true);
+
+  const activeTechnology = useSelector(({ chrome: { activeTechnology } }) => activeTechnology);
+  const appId = useSelector(({ chrome: { appId } }) => appId);
+  const globalNav = useSelector(({ chrome: { globalNav } }) => globalNav, globalNavComparator);
```

Comment:
The chrome could be undefined, can you change it to safe accessor?
```suggestion
  const activeTechnology = useSelector(({ chrome }) => chrome?.activeTechnology);
  const appId = useSelector(({ chrome }) => chrome?.appId);
  const globalNav = useSelector(({ chrome }) => chrome?.globalNav, globalNavComparator);
```

Reply:
I am not sure that it could be undefined, when it's defined on redux init.

Reply:
The page with not logged in account has empty chrome and it caused a lot of problems in the past. I'd rather avoid it with the safe accessors.